dev-libs/sway/sway-9999.ebuild

```diff
@@ -94,7 +92,9 @@ src_configure() {
 pkg_postinst() {
 	elog "You must be in the input group to allow sway to access input devices!"
 	if ! use systemd && ! use elogind; then
-		fcaps_pkg_postinst
+		local root=${EROOT:-${ROOT}}
+		chown root:0 "${root%/}/usr/bin/sway" || die
+		chmod 4711 "${root%/}/usr/bin/sway" || die
```

Comment:
and fperms here?

https://devmanual.gentoo.org/function-reference/install-functions/